src/service-worker/core.js

```diff
@@ -624,6 +624,10 @@ export function handleFetch(request, maybeClientId) {
       // If not, let's fetch and cache the request.
       return fetchJsFile(cache, versionedRequest, pathname, version);
     });
+  }).catch(err => {
+    // Throw error out of band.
+    Promise.reject(err);
```

Comment:
That's exactly what it does 😉. And because I don't attach a `#catch` block to it, its rejection won't ever be handled.

I could have also done a `setTimeout` and `throw`.